pkg/resources/password_policy.go

```diff
@@ -90,6 +90,13 @@ var passwordPolicySchema = map[string]*schema.Schema{
 		Description:  "Specifies the minimum number of special characters the password must contain. Supported range: 0 to 256, inclusive. Default: 1",
 		ValidateFunc: validation.IntBetween(0, 256),
 	},
+	"min_age_days": {
+		Type:         schema.TypeInt,
+		Optional:     true,
+		Default:      0,
```

Comment:
i don't know that we actually need to set the default. Snowflake has its own default if a value isn't set, we don't need to specify it here.

Reply:
The default is set for other params in this resource. The default is the SF default from the docs. I have added it for two reasons:
- consistency with other parameters
- consistency in update implementation for this resource (as you can see, there is no set/unset, but set only; it behaves right because `d.Get()` returns the default if not set - I added test case for this because the implementation seemed strange for me at first sight)